recipes/glog/all/conanfile.py

```diff
@@ -68,7 +68,12 @@ def _configure_cmake(self):
         self._cmake.definitions["WITH_THREADS"] = self.options.with_threads
         if tools.Version(self.version) >= "0.5.0":
             self._cmake.definitions["WITH_PKGCONFIG"] = True
-            self._cmake.definitions["WITH_SYMBOLIZE"] = True
+            if self.settings.os == "Emscripten":
+                self._cmake.definitions["WITH_SYMBOLIZE"] = False
+                self._cmake.definitions['HAVE_SYSCALL_H'] = "0"
+                self._cmake.definitions['HAVE_SYS_SYSCALL_H'] = "0"
```

Comment:
Please keep the recipe consistent.
```suggestion
                self._cmake.definitions["HAVE_SYSCALL_H"] = False
                self._cmake.definitions["HAVE_SYS_SYSCALL_H"] = False
```

Reply:
https://github.com/conan-io/conan-center-index/pull/7126/commits/31451991e6173b23d8d1386e25a8f6d8a85d8cac